cmd/kyma/install/cmd.go

```diff
@@ -54,15 +52,15 @@ The standard installation uses the minimal configuration. The system performs th
 
```

Comment:
Do I understand correctly, the system performs only two "main" steps, but depending on the installation type, main step 1 has two different sets of 6 substeps, of which the last 3 are identical for both types?  Can we elevate those last 3 substeps (overrides, admin pw, minikube patch) to main step level and only keep the ones that are different below the introduction "the following steps differ..."